src/pubkey.cpp

```diff
@@ -188,6 +188,18 @@ bool XOnlyPubKey::CheckPayToContract(const XOnlyPubKey& base, const uint256& has
     return secp256k1_xonly_pubkey_tweak_add_check(secp256k1_context_verify, m_keydata.begin(), parity, &base_point, hash.begin());
 }
 
+bool CPubKey::TweakMulVerify(const CPubKey& untweaked, const uint256& tweak) const
+{
+    assert(this->IsCompressed() && untweaked.IsCompressed());
```

Comment:
I'm not sure that `untweaked.IsCompressed()` is a precondition.

Reply:
Agreed, no need to assert this, but for future reviewers, be aware that the sole use of this function does actually enforce that `untweaked` is compressed.